### PR TITLE
Fixes missing autoStart documentation

### PR DESCRIPTION
**Pull request by xavierfoucrier**
Adds a base description for the `autoStart` parameter in the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ You can modify the options for the plugin by passing in JSON upon instantiation.
 | cursorChar    | (string) The character used for the cursor. HTML works too! | pipe |
 | breakLines    | (boolean) Choose whether you want multiple strings to be printed on top of each other (`breakLines: true`), or if you want each string to be deleted and replaced by the next one (`breakLines: false`).  | true  |
 | nextStringDelay    | (number in milliseconds or array) The amount of time (milliseconds) between typing the next string when multiple strings are defined. You may either pass a number in milliseconds, or an array of values. The first value will be used as the delay before a new string starts, and the second value will be used as the delay after a string ends. For example, passing `[1000, 2000]` will tell TypeIt to pause 1000ms before typing a new string, and wait 2000ms after a string has just completed. | 750 |
+| autoStart    | (boolean) Defines if the instance start typing automatically. Turn this option to `false` if you want to create chain strings with the `type()`, `delete()` or `pause()` methods.  | true  |
```
Comment (alexmacarthur) [inline README.md]: I don't think the "Turn this option `false`" sentence is needed -- `autoStart` should work the same regardless of how you're queuing up strings. Is this not how it works for you?

**Comment by alexmacarthur** [comment]
@xavierfoucrier Thanks for the PR!! One question about the second sentence you added. I'm seeing `autoStart` work for me no matter how I define strings/commands. Let me know if you're seeing otherwise. If you are, that's a bug I should fix.

**Comment by xavierfoucrier** [comment]
Hi @alexmacarthur!
Thanks for your feedback, I will check again and get back to you quickly.

**Comment by xavierfoucrier** [comment]
In fact, there is no effect using the `autoStart` parameter with `type()`, `delete()` or `pause()` method: I wrote this based on [your example](https://github.com/alexmacarthur/typeit#chaining-on-initializing) in the documentation... but, I think I found a bug when used in the context below: can't figure out if it's normal.

```
<p class="typeit">Hello, how I can help you?</p>
...
new TypeIt('.typeit', {
  autoStart: false
});
```

With this code, the Typewriter will start typing automatically: may be its because there is already some text inside the paragraph... or maybe I misunderstood the aim of this parameter :thinking:

**Comment by alexmacarthur** [comment]
Ah.. I think I know what's going on. `autoStart` will tell the instance to start typing only when the element is in the viewport. If you set it to `true`, it will start no matter where it is, but if it's set to `false`, it will start only when a user scrolls to the instance. I'm not near a computer now but will verify that when I am. Sorry for the confusion... That's perhaps an API update I need to make. I can see that confusing people.

**Comment by xavierfoucrier** [comment]
Oh! OK... it's very confusing, now I understand the logic.

I suggest you to change the name of this settings with a more consistent name, like **autoStartInViewport**, **startInViewport** or simply **typeInViewport**.

I will update my PR based on your detailed comments for now, so you can update your API later.

**Pull request merged**